server/controllers/CommentController.js

```diff
@@ -107,13 +112,14 @@ exports.delete = async (req, res) => {
   res.status(204).send();
 };
 
-async function reply_to_comment(parentId, text, userId) {
+async function reply_to_comment(parentId, text, userId, runId = "") {
```

Comment:
Is there a reason why this function is not camelCase?

Reply:
I wrote it in snake case to distinguish it from the controllers in the file. This is only meant to be used as a helper function.